lerobot/common/policies/diffusion/modeling_diffusion.py

```diff
@@ -304,7 +304,11 @@ def compute_loss(self, batch: dict[str, Tensor]) -> Tensor:
         loss = F.mse_loss(pred, target, reduction="none")
 
         # Mask loss wherever the action is padded with copies (edges of the dataset trajectory).
-        if self.config.do_mask_loss_for_padding and "action_is_pad" in batch:
+        if self.config.do_mask_loss_for_padding:
+            if "action_is_pad" not in batch:
+                raise ValueError(
+                    "You need to provide 'action_is_pad' in the batch when config.do_mask_loss is set."
```

Comment:
Suggestion, fix parameter name (was missing `_for_padding`) and while we're at it, use formatting to improve maintainability (now the variable name is correct, and will be tracked with refactors). FYI I don't always do the formatting trick, just when it's an easy win.

```suggestion
                    f"You need to provide 'action_is_pad' in the batch when {self.config.do_mask_loss_for_padding=}."
```

Reply:
makes a lot of sense! :slightly_smiling_face: :raised_hands: